lib/new_relic/agent/instrumentation/aws_sqs.rb

```diff
@@ -0,0 +1,25 @@
+# This file is distributed under New Relic's license terms.
+# See https://github.com/newrelic/newrelic-ruby-agent/blob/main/LICENSE for complete details.
+# frozen_string_literal: true
+
+require_relative 'aws_sqs/instrumentation'
+require_relative 'aws_sqs/chain'
+require_relative 'aws_sqs/prepend'
+
+DependencyDetection.defer do
+  named :aws_sqs
+
+  depends_on do
```

Comment:
Where are we checking the config value `instrumentation.aws_sqs` to know if we should be instrumenting? I feel like I usually see it here

Reply:
the `named :aws_sqs` is what tells dependency detection to check the `instrumentation.aws_sqs` config value. we don't usually have to check it ourselves (unless we have multiple, or its more complicated for some reason)

Reply:
Thank you for going over this!